bgpd/bgp_vpn.c

```diff
@@ -228,16 +228,6 @@ int show_adj_route_vpn(struct vty *vty, struct peer *peer,
 							  use_json, json_array);
 				}
 			}
-			if (use_json && rm) {
```

Comment:
I'm not certain this block should be removed entirely -- I think @donaldsharp or someone else more familiar with the json code should review here... It seem that just the `&& rm` should be removed, rather than the entire block?

Reply:
I think this if statement should be moved up to line 226. with a if (use_json) { <code from 232-239> } else route_vty_out_tmp(vty,...)